content/docs/docs-nav.yaml

```diff
@@ -47,9 +47,9 @@ nav:
     - Artifactory: '/docs/custom-plugins/artifactory/'
     - Configuring: '/docs/custom-plugins/configuring/'
     - Authenticated Proxy: '/docs/custom-plugins/authenticated-proxy/'
-  - Details:
-    - GitHub app permissions: '/docs/integrations/github-app-permissions/'
+  - In-depth:
+    - GtHub app permissions: '/docs/details/github-app-permissions/'
     - Setting secrets: '/docs/details/setting-secrets/'
-    - Updating the UI: '/docs/getting-started/updating-the-ui/'
+    - Updating the UI: '/docs/details/updating-the-ui/'
     - Allowlisting Roadie Traffic: '/docs/details/allowlisting-roadie-traffic/'
-    - Required annotations: '/docs/integrations/required-annotations'
```

Comment:
I'm changing these URLs so I can make a simple rule that all URLs with "integrations" in them can go into the "Plugins and integrations" tab and all URLs with "details" in them can go in the "In-depth" tab.

Redirects for the old URLs are handled in the netlify.toml below.